### DISCUSSION TITLE
ip suddenly disappears while running miner.

### DISCUSSION THREAD

**Discussion by zzangtrish** [filecoin-project/lotus#5738] [General Q&A] [answered]
The master Listen IP the worker is seeing suddenly disappears.

Looking at ifconfig -a, 10G LAN card recognition improves, but ip suddenly disappears from the device name and vi / etc / netpan / device names are the same.

The ip does not come up even if I proceed with netplan apply, and the ip comes up after rebooting the server itself.

Has anyone ever experienced such a problem?

**Comment by stuberman** [comment]
I recently had some problem with IP addresses on my Ubuntu server. If I remember correctly the name of the interface would change and lose its setting. I had to set the interface name based on MAC address - not sure if this is related to your problem.

This is the new config:
`sudo nano /etc/netplan/00-installer-config.yaml`

> network:
>   ethernets:
>     eth_1gb:
>       match:
>         macaddress: 3c:7c:3f:20:fe:a7
>       set-name: eth_1gb
>       addresses:
>         - 192.168.1.96/24
>       gateway4: 192.168.1.254
>       nameservers:
>           search: [att.local]
>           addresses: [8.8.8.8, 8.8.4.4]
>   version: 2

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
we have changed network names if we change hardware PCIe configurations.

maybe have a look at

https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/
